server/routes/index.ts

```diff
@@ -0,0 +1,21 @@
+import type { RequestHandler, Router } from 'express'
+
+import asyncMiddleware from '../middleware/asyncMiddleware'
+
+export function routes(router: Router): Router {
+  const get = (path: string | string[], handler: RequestHandler) => router.get(path, asyncMiddleware(handler))
+
+  get('/', (req, res) => res.redirect('/calendar#today'))
+  get('/contact-us', (req, res) => {
+    res.render('pages/contact-us', {
```

Comment:
seemed more sensible to move the contact us code here rather than creating a controller etc.